detection_rules/ecs.py

```diff
@@ -98,7 +99,7 @@ def get_max_version(include_master=False):
     versions = get_schema_map().keys()
 
     if include_master and any([v.startswith('master') for v in versions]):
-        return glob.glob(os.path.join(ECS_SCHEMAS_DIR, 'master*'))[0]
+        return list(ECS_SCHEMAS_DIR.glob('master*'))[0]
```

Comment:
@brokensound77 do you know if this will return the desired path if there are multiple directories with the prefix, `master`?

Reply:
There should never be concurrent versions that start with `master`, so long as schemas are only added with the `download_schemas` function, because of line 262, which removes it before building a new one